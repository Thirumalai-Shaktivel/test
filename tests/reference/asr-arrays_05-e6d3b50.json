{
    "basename": "asr-arrays_05-e6d3b50",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_05.f90",
    "infile_hash": "09eac1191b2e8a76fafedf1307b2d1240ffbc5f632c08afa847df805",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-arrays_05-e6d3b50.stdout",
    "stdout_hash": "bcb7597d9fd015ef16073e816326a691768ef6096f9f87f6bda289b2",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}