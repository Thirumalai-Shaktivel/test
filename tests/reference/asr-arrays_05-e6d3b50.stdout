(TranslationUnit (SymbolTable 1 {arrays_05: (Program (SymbolTable 2 {i: (Variable 2 i Local () () Default (Integer 4 []) Source Public Required .false.), numbers: (Variable 2 numbers Local (ConstantArray [(ConstantReal 1.500000 (Real 4 [])) (ConstantReal 3.200000 (Real 4 [])) (ConstantReal 4.500000 (Real 4 [])) (ConstantReal 0.900000 (Real 4 [])) (ConstantReal 7.200000 (Real 4 []))] (Real 4 [])) () Default (Real 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])))]) Source Public Required .false.)}) arrays_05 [] [(DoLoop ((Var 2 i) (ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])) ()) [(= (ArrayRef 2 numbers [(() (Var 2 i) ())] (Real 4 [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 5 (Integer 4 [])))]) ()) (BinOp (ImplicitCast (Var 2 i) IntegerToReal (Real 4 []) ()) Mul (ConstantReal 2.000000 (Real 4 [])) (Real 4 []) () ()) ())]) (= (Var 2 numbers) (ConstantArray [(ConstantReal 1.500000 (Real 4 [])) (ConstantReal 3.200000 (Real 4 [])) (ConstantReal 4.500000 (Real 4 [])) (ConstantReal 0.900000 (Real 4 [])) (ConstantReal 7.200000 (Real 4 []))] (Real 4 [])) ())])}) [])
